onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -107,34 +110,53 @@ def __init__(
             else extra_options["QDQOpTypePerChannelSupportToAxis"]
         )
 
-    def quantize_tensor(self, tensor_name):
+    def _is_tensor_quantizable(self, tensor_name):
```

Comment:
```suggestion
    def _is_tensor_quantizable(self, tensor_name: str) -> bool:
```

Reply:
will have a separate PR to update the type annotations for all functions under quantization.